### PR TITLE
Update a few things on the website.

### PR DESCRIPTION
**Pull request by AlessGoss**

### CODE REVIEW

**Comment by windowz414** [comment]
Thank you for the contribution! Just merged the commits in!

**Review by windowz414 — COMMENTED** [review]
Seems all good, but make sure to use the `Fixes:` tag on your commits and sign them off (`git commit -s`) whenever you can.

For a little demonstration, let's see how you can do so;

First, before we edit something broken in your current HEAD, we first use `git-blame` to see which line was committed with which commit, on index.html for example;
```
git blame index.html
```

In here, you should find the faulty line, and copy its commit hash which will at the very beginning of that line viewed in the blame. If you prefer to have a full commit hash instead, unshallow your repo if you had it cloned with the `--depth=X` option and do a `git log` and search for the hash you copied from blame.

After that, when you're writing your commit message, append this right before the `Signed-off-by:` tag like this;
```
Fixes: <commit hash>
```

Another thing I would like to mention is to avoid usage of periods (.) at the very end of commit titles. Almost all open source projects don't prefer doing this - Think of it like an essay title, you wouldn't put a period at the end of a title.

Also append filename at the beginning of your commit titles, with shortened paths if under a subdirectory. It will greatly help to keep track of files. Don't append file name only if the diff is too big.

Lastly, please make sure to explain what a commit does (don't well-document it, but also don't leave it commentless). This is essential to keep track of what changed without trying to open up the commit.

So, in short, your commit messages should look like this;
```
filename: Short change description

This is a long change description, not well-documented but long enough
for someone new to the language of this repo to understand what is
going on, to explain what has changed in a deeper way. The message is
indented with 72 character limit as per Linux Kernel's commit
procedures as well!

# Append "Fixes:" tag here if it fixes a commit. Example;
# Fixes: 363138152d34689467cbad3cc0e26d0bde1e28ac
Signed-off-by: Beru Hinode <windowz414@1337.lgbt>
```

But still, good job overall! Even though these are minor, I just couldn't take time to spot and fix it all! Thanks a lot!

**Comment by AlessGoss** [comment]
Alright. I don't have much experience with that, but I'll try to do so instead. :)
Also, you're welcome. ;)
(By the way, I'm using GitHub Desktop instead of Git.)

**Comment by windowz414** [comment]
> Alright. I don't have much experience with that, but I'll try to do so instead. :) Also, you're welcome. ;) (By the way, I'm using GitHub Desktop instead of Git.)

IIRC, GitHub Desktop doesn't have support for signing off directly. But use it only if it has support for it. Also make sure you have a GPG key with your verified GitHub email and your Git config name yourself since it's used to sign off the commits.

**Comment by windowz414** [comment]
One other thing I would like to touch...

> `Author: Aless <54534326+AlessGoss@users.noreply.github.com>`

Please, PLEASE, avoid using GitHub's pseudo email addresses. It can be disastrous when someone wants to contact you for a change you did or add you to CC lists during a conversation as an outsider collaborator or simply when identifying your identity with GPG and/or `git-log`/`git-blame`. Use a real email address wherever possible.

**Comment by AlessGoss** [comment]
> IIRC, GitHub Desktop doesn't have support for signing off directly. But use it only if it has support for it.

I checked GitHub Desktop settings and it still doesn't seem to have support for it.

> Use a real email address wherever possible.

I think I would do that later, because I'm still afraid to reveal any of my email addresses. But if I would use one, it would be my contact email address.

**Comment by windowz414** [comment]
> > Use a real email address wherever possible.
> 
> I think I would do that later, because I'm still afraid to reveal any of my email addresses. But if I would use one, it would be my contact email address.

I think you can use ProtonMail at least, like Nick and many other developers do, no privacy leakage at all.

Or a custom email address if possible, just like I do.

**Comment by AlessGoss** [comment]
> I think you can use ProtonMail at least, like Nick and many other developers do, no privacy leakage at all.

I would use Gmail, but yeah, that could also be a good idea. :)

> Or a custom email address if possible, just like I do.

Since I don't have access to a domain name, I don't think that's possible.

**Comment by windowz414** [comment]
> I would use Gmail, but yeah, that could also be a good idea. :)

There you have it now! :)

**Comment by AlessGoss** [comment]
> There you have it now! :)

Yeah. :)
